routes/donations.js

```diff
@@ -82,9 +139,17 @@ router.post('/', async (req, res) => {
         email,
         phoneNum,
         notes,
-        date,
+        submittedDate,
       },
     );
+    // console.log(donation);
+    pictures.map(async ({ imageUrl, picNotes }) => {
+      await insertPicture(donation[0].id, imageUrl, picNotes);
+    });
```

Comment:
follow the same naming scheme as shown in `insertPictures` and the database model for argument names